Raspi/Test/mqtt_wireless_log.py

```diff
@@ -0,0 +1,152 @@
+import paho.mqtt.client as mqtt
+import os
+import pandas as pd
+from DataToTempCSV import DataToTempCSV
+import json
+from datetime import datetime
+
+
+def on_connect(client, userdata, flags, rc):
+    """ When the MQTT client connects to the broker it prints out if it
+    connects successfully and then subscribes to all of the wireless-module
+    topics"""
+
+    if rc == 0:
+        print("Connected Successfully! Result code: " + str(rc))
+    else:
+        print("Something went wrong! Result code: " + str(rc))
+
+    # Subscribe to all of the wireless module topics
+    client.subscribe("/v3/wireless-module/#")
+
+
+def on_message(client, userdata, msg):
+    """ MQTT callback for when data is sent on the subscribed
+    '/v3/wireless-module/#' topics. The module_id is found by squashing M
+    infront of the module number. eg M1, M2, M3... etc. The module_id is used
+    for identifying what data came from where and is also used for naming the
+    temp files.
+    """
+
+    module_id = "M" + str(msg.topic[20])
+
+    # Start the recording of <module_id>
+    if msg.topic.endswith("start"):
+        # Clear the old temporary files in the folder, just in case the script
+        # exited early and they were not cleared properly
+        remove_files(find_temp_csvs(module_id))
+
+        start_data = msg.payload.decode("utf-8")
+        start_data = json.loads(start_data)
+
+        # Save the state of recording and the output filename to global dicts
+        is_recording[module_id] = True
+        module_start_time[module_id] = datetime.now()
+        output_filename[module_id] = start_data["filename"]
+
+        print(module_id, "STARTED")
+
+    # Stop the recording of <module_id>
+    elif msg.topic.endswith("stop"):
+        # Change the state of recording to false in global dict
+        is_recording[module_id] = False
+
+        print(module_id, "STOPPED")
+
+        # Save the temp CSV data into a proper CSV
+        save_temp_csv(module_id)
+
+    # Record low battery data
+    elif msg.topic == "/v3/wireless-module/battery/low":
+        module_data = msg.payload.decode("utf-8")
+        module_data = json.loads(module_data)
+        module_id = "M" + str(module_data["module-id"])
+
+        DataToTempCSV(msg, module_start_time[module_id], module_id)
+
+    # Record other data (battery and sensor data)
+    elif is_recording[module_id] is True:
```

Comment:
Since `is_recording[module_id]` is a boolean (more specifically it is [truthy](https://stackoverflow.com/questions/39983695/what-is-truthy-and-falsy-how-is-it-different-from-true-and-false)) you can drop the `is True` part.